bn-models/src/models/wcgop/wcgop-operation.ts

```diff
@@ -17,58 +55,185 @@ declare type ProductDeliveryState = any; // TODO Lookup
 
 // TODO: Create both WCGOP Trawl and Fixed Gear haul types?
 export interface WcgopOperation extends BaseOperation {
-  biolist?: number;
-  catches?: WcgopCatch[];
-  operationNum?: number; // Sequential
-  locations?: FishingLocation[];
+  /*
+  Attributes from extended interfaces are accessible in this interface.
+
+  WcgopOperattion <-- BaseOperation <-- Base
+
+      Base attributes:
+        _id
+        _rev
+        type
+        createdBy
+        createdDate
+        updatedBy
+        updatedDate
+        uploadedDate
+        notes
+        dataSource
+        isDeleted
+        legacy
+        changeLog
+
+      BaseOperation attributes:
+        None
+  */
+  biolist?: number; // randomly selected list of protocols for biospecimens (Trawl: 1-3, random; FG: 4/5, based on depth)
+  catches?: WcgopCatch[]; // subdocs w. whats caught; stores taxonomy, weights, counts, biospecs, dissections etc
+  operationNum?: number; // sequential, ordered based on retrieval time of haul/set
+  locations?: FishingLocation[]; // lat/lon/depth/time; typically two locations (gear set and gear picked up), FG sets may have more
   observerTotalCatch?: {
-    measurement?: Measurement;
-    weightMethod?: WeightMethod;
+    // observers rough haul weight; not an expanded value, anecdotal
+    measurement?: Measurement; // weight and units (always in lbs)
+    weightMethod?: WeightMethod; // method observer used; commonly Visual Estimate (14) or Retained + Discard (11)
   };
-  gearType?: GearType;
-  gearPerformance?: GearPerformance;
-  targetStrategy?: WcgopTargetStrategy;
-  isEfpUsed?: boolean;
-  calWeight?: number;
-  fit?: number;
-  isOperationUnsampled?: boolean;
-  isGearLost?: boolean;
-  isDataQualityPassing?: boolean;
-  sightingEventIds?: CouchID[];
+  gearType?: GearType; // gear used, trawl or FG-specific (e.g. Longline, troll, pot, groundfish trawl etc)
+  gearPerformance?: GearPerformance; // any issues with gear? No Problem, Problem-net hung up, Problem...
+  targetStrategy?: WcgopTargetStrategy; // grouping of fish targeted (e.g. DOVR, SABL etc)
```

Comment:
see you documented this in your questions, just thinking once we know the values we could create an enum for the different wcgopTargetStrategies.